src/js/game.js

```diff
@@ -0,0 +1,195 @@
+import { ERROR_MESSAGE, CONGRATS_MESSAGE, INPUT_TYPE } from "./constants.js";
+import { $ } from "./utils.js";
+
+export default function CarGame() {
+  const form = $("form");
+  const names = $("#names");
+  const namesInput = $("#names input");
+  const nameCards = $("#nameCards");
+  const nameSubmitBtn = names.querySelector("button");
+  const gameWinner = $("#winner");
+
+  let carNames = [];
+  let times = "";
+  let carsDistance = [];
+  let winner = [];
+
+  namesInput.addEventListener("keyup", ({ target }) => {
+    carNames = target.value;
+    carNames = checkCarNames(carNames);
+  });
+
+  function onCarNamesSubmit() {
+    nameSubmitBtn.addEventListener("click", () => {
+      showGameTimesInput();
+      onGameTimesSubmit();
+      preventInput(INPUT_TYPE.NAME);
+    });
+  }
+
+  function checkCarNames(names) {
+    const namesArr = names.split(",");
+    const nameBlankCheck = namesArr
+      .map(name => name.replace(/(^\s*)|(\s*$)/gi, ""))
+      .filter(name => !!name);
+    const nameLengthCheck = nameBlankCheck.map(x => isNameUnderFiveWords(x));
+    return nameLengthCheck;
+  }
+
+  function isNameUnderFiveWords(name) {
+    const isUnderFiveWords = /^.{0,5}$/;
+    if (isUnderFiveWords.test(name)) {
+      return name;
+    } else {
+      alert(ERROR_MESSAGE.NAME_LENGTH_ERROR);
+      return name.slice(0, 5);
+    }
+  }
+
+  function preventInput(type) {
+    if (type === INPUT_TYPE.NAME) {
+      namesInput.disabled = true;
+      nameSubmitBtn.disabled = true;
+      return;
+    }
+
+    if (type === INPUT_TYPE.GAMETIMES) {
+      const gameTimes = $("#times");
+      const gameTimesInput = $("#times input");
+      const gameTimesSubmitBtn = gameTimes.querySelector("button");
+      gameTimesInput.disabled = true;
+      gameTimesSubmitBtn.disabled = true;
+      return;
+    }
+  }
```

Comment:
mapper를 둬서 if문 블록을 줄이는 방법 추천드려봅니다!

```suggestion
  const MAPPER = {}
  MAPPER[INPUT_TYPE.NAME] = disableNames()
  MAPPER[INPUT_TYPE. GAMETIMES] = disableGameTimes()
  
  function disableNames() {
    namesInput.disabled = true;
    nameSubmitBtn.disabled = true;
  }
  
  function disableGameTimes() {
    const gameTimes = $("#times");
    const gameTimesInput = $("#times input");
    const gameTimesSubmitBtn = gameTimes.querySelector("button");
    gameTimesInput.disabled = true;
    gameTimesSubmitBtn.disabled = true;
  }
  
  function preventInput(type) {
    MAPPER[type]()
  }
```